src/state/collectibles/index.ts

```diff
@@ -20,15 +20,25 @@ export const fetchWalletNfts = createAsyncThunk<NftSourceItem[], string>(
   async (account) => {
     // For each nft source get nft data
     const nftSourcePromises = Object.keys(collections).map(async (nftSourceType) => {
+      // Temporarily skip over pancake-squad
+      if (collections[nftSourceType].slug === 'pancake-squad') {
+        return []
+      }
       const { address: addressObj } = collections[nftSourceType]
+
       const address = getAddress(addressObj)
       const contract = getErc721Contract(address)
+      const balanceOfResponse = await contract.balanceOf(account)
+      const balanceOf = balanceOfResponse.toNumber()
+
+      if (balanceOf === 0) {
```

Comment:
Can we fix this by just using the Etheres BN `balanceOfResponse.eq(0)`

Reply:
The `balanceOf` value (as a number) is used as part of the for loop further down - so I'll leave the `.toNumber()` call in, but sure this condition could be modified

Reply:
Added